code/game/objects/structures/icemoon/cave_entrance.dm

```diff
@@ -152,41 +160,93 @@ GLOBAL_LIST_INIT(ore_probability, list(
+			new /obj/structure/cursed_slot_machine(loc)
+			new /obj/item/stack/spacecash/c1000(loc)
+			new /obj/item/stack/spacecash/c1000(loc)
+			new /obj/item/coin/gold(loc)
+			new /obj/item/coin/gold(loc)
+			new /obj/item/coin/gold(loc)
+			new /mob/living/simple_animal/hostile/faithless(loc)
 		if(15)
 			new /obj/item/ship_in_a_bottle(loc)
 			new /obj/item/oar(loc)
+			new /obj/item/clothing/under/costume/sailor(loc)
+			new /mob/living/simple_animal/hostile/carp/megacarp(loc)
 		if(16)
 			new /obj/item/seeds/gatfruit(loc)
+			new /obj/item/seeds/cannabis(loc)
+			new /obj/item/cultivator/rake(loc)
+			new /mob/living/simple_animal/hostile/venus_human_trap(loc)
+			new /mob/living/simple_animal/hostile/venus_human_trap(loc)
 		if(17)
 			new /obj/item/reagent_containers/food/drinks/drinkingglass/filled/nuka_cola(loc)
+			new /obj/item/reagent_containers/food/drinks/drinkingglass/filled/nuka_cola(loc)
+			new /obj/item/reagent_containers/food/drinks/drinkingglass/filled/nuka_cola(loc)
+			new /obj/item/stack/sheet/mineral/uranium/twenty(loc)
 		if(18)
 			new /obj/item/soulstone/anybody(loc)
+			new /obj/item/soulstone/anybody(loc)
+			new /obj/structure/constructshell(loc)
+			new /mob/living/simple_animal/hostile/construct/juggernaut(loc)
 		if(19)
-			new /obj/item/disk/design_disk/modkit_disc/resonator_blast(loc)
+			if(prob(50))
+				new /obj/item/disk/design_disk/modkit_disc/resonator_blast(loc)
+			else
+				new /obj/item/disk/design_disk/modkit_disc/rapid_repeater(loc)
+			new /obj/effect/mob_spawn/human/miner(loc)
 		if(20)
-			new /obj/item/disk/design_disk/modkit_disc/rapid_repeater(loc)
+			new /obj/effect/rend(loc)
+			new /obj/item/kitchen/knife/bloodletter(loc)
+			new /mob/living/simple_animal/hostile/alien/maid(loc)
 		if(21)
-			new /obj/item/slimepotion/transference(loc)
+			new /mob/living/simple_animal/hostile/big_legion(loc)
+			new /obj/item/organ/tongue/bone(loc)
+			new /obj/structure/closet/crate/necropolis/tendril(loc)
 		if(22)
 			new /obj/item/slime_extract/adamantine(loc)
+			new /obj/item/slimepotion/slime/sentience(loc)
+			new /obj/item/slimepotion/transference(loc)
+			new /obj/effect/mob_spawn/human/scientist(loc)
 		if(23)
 			new /obj/item/weldingtool/abductor(loc)
+			new /obj/item/abductor_machine_beacon/chem_dispenser(loc)
+			new /obj/structure/fluff/iced_abductor(loc)
 		if(24)
 			new /obj/structure/elite_tumor(loc)
 		if(25)
 			new /mob/living/simple_animal/hostile/retaliate/clown/clownhulk(loc)
 		if(26)
 			new /obj/item/clothing/shoes/winterboots/ice_boots(loc)
+			new /mob/living/simple_animal/hostile/bear/snow(loc)
+			new /obj/effect/decal/remains/human(loc)
 		if(27)
 			new /obj/item/book/granter/spell/sacredflame(loc)
+			new /obj/effect/temp_visual/fire(loc)
 		if(28)
 			new /mob/living/simple_animal/hostile/megafauna/blood_drunk_miner/doom(loc)
+		if(29)
+			new /mob/living/simple_animal/hostile/megafauna/swarmer_swarm_beacon(loc)//god help you
+		if(30)
```

Comment:
All of these right here are a hard no

Reply:
The swarmer beacon now drops a small stack of swarmers and an infiltrator engi, and the syndicate now has if/either logic for either reward